src/components/TodoList.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+ 
+    
+            <!-- <div >
+                
+            <button class="btn" v-for="post in posts" v-bind:title="post.data" v-bind:key="post.id" 
+            v-bind:style = '{"text-decoration" : (post.isCompleted? "line-through" : "none" )}' 
+            @click="post.isCompleted = !post.isCompleted">
+                
+                {{ post.data }}
+                
+               
+            </button>
+             </div> -->
+            
+       
+
+        
+        <div class="div">
+            
+            <div class="inp">
+                <input type="text" v-model="posts.data" placeholder="Type in something">
```

Comment:
Seems like the v-model here should be something like `newTodo`. Why are you using `.data` here and the line below?

Reply:
When I change the v-model to `newTodo` (or any other name) it gives me an error message like:
> "Property or method "newTodo" is not defined on the instance but referenced during render."


The error seems to disappear only when I use `posts.data` in the v-model and the line below. Should I be concerned about the error because it does not seem to break the app or anything like that

Reply:
According to the examples [here](https://vuejs.org/v2/guide/forms.html), it looks like if a variable like `newTodo` is set in the `data` part of the Vue instance, `v-model` should reference it as `newTodo`.